hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -63,6 +71,18 @@ public void enrichEvent(StructuredTrace trace, Event event) {
                       }
                       return attributeValue;
                     }));
+
+        // Add deployment attribute if pod name is available
+        if (resourceAttributeKey.equals(POD_NAME_KEY)) {
+          resourceAttributeMaybe.ifPresent(
+              attributeValue ->
+                  attributeMap.computeIfAbsent(
+                      DEPLOYMENT_KEY,
```

Comment:
Resource Attributes to add are controlled by the `ResourceAttributeEnricher` config present in the `application.conf` file, so that we can enable/disable any resource attribute enrichment from the config. 
Add `deployment.type` in the config and the logic as well to only enrich attributes only if it is present in the config.